packages/components/molecules/f-media-element/CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v0.3.1
+------------------------------
+*June 21, 2021*
+
+### Added
+- stackOnMobile prop to stack when on device sizes <=narrowMid
```

Comment:
Should reference width by `narrow`/`mid` etc, as that's what we're checking for (someone could have a narrow screen on tablet/desktop for instance). So maybe `stackWhenNarrow`?

Reply:
`stackWhenNarrow` seemed a good choice